packages/components/src/components/DaCard.vue

```diff
@@ -226,13 +226,14 @@ export default {
       box-shadow: 0 16px 24px 0 rgba(0, 0, 0, 0.64);
       transition: opacity 0.4s ease-in-out;
     }
+  }
 
-    & .hover, :hover {
-      transform: translateY(-4px);
```

Comment:
initial fix was able to restrict animation based on configuration but this style is not capable of performing the animation when allowed.  had to explicitly mention the `.card` selector.